qutebrowser/misc/utilcmds.py

```diff
@@ -42,15 +42,17 @@
 
 @cmdutils.register(maxsplit=1, no_cmd_split=True, no_replace_variables=True)
 @cmdutils.argument('win_id', value=cmdutils.Value.win_id)
-def later(ms: int, command: str, win_id: int) -> None:
+def later(duration: str, command: str, win_id: int) -> None:
     """Execute a command after some time.
 
     Args:
-        ms: How many milliseconds to wait.
+        duration: Duration to wait in format XhYmZs or number for seconds.
```

Comment:
```suggestion
        duration: Duration to wait in format XhYmZs or a number for milliseconds.
```

(milliseconds, not seconds, right?)